prow/plugins/config.go

```diff
@@ -621,6 +622,18 @@ func (c *Configuration) TriggerFor(org, repo string) Trigger {
 	return Trigger{}
 }
 
+// CurrentMilestoneFor finds the current milestone for a repo, if one exists
+// a milestone can be listed for the repo itself or for the
+// owning organization. This is used by the milestoneapplier plugin.
+func (c *Configuration) CurrentMilestoneFor(org, repo string) string {
+	for r, milestone := range c.MilestoneApplier {
+		if r == org || r == fmt.Sprintf("%s/%s", org, repo) {
```

Comment:
If we want to avoid the repo-level override problem (https://github.com/kubernetes/test-infra/issues/9757), I think limiting this to just org/repo should also be ok because each repo will mostly have different milestones, and it's sort of rare that all repos in an org follow the same milestone. Thoughts?

cc @stevekuznetsov

Reply:
per-repo sgtm.